frontend/src/components/ConfirmForgotPassword.js

```diff
@@ -0,0 +1,150 @@
+import { withStyles } from '@material-ui/core/styles'
+import { Button, Grid, Paper, TextField, Typography } from '@material-ui/core'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { forgotPasswordSubmit, forgotPassword } from '../actions/auth'
+import { Redirect } from 'react-router-dom'
+import ErrorMessage from './ErrorMessage'
+
+const styles = (theme) => ({
+  root: {
+    background: 'linear-gradient(to right, #5e60ce, #5390d9)',
+    display: 'flex',
+    flexDirection: 'column',
+    minWidth: '100%',
+    minHeight: '100vh',
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  paper: {
+    minWidth: '340px',
+    padding: theme.spacing.unit * 2
+  },
+  input: {
+    width: '100%'
+  },
+  button: {
+    width: '100%',
+    marginTop: theme.spacing.unit
+  }
+})
+
+class ConfirmForgotPassword extends Component {
+  state = {
+    email: this.props.auth.email,
+    confirmationCode: '',
+    newPassword: ''
+  };
+
+  validate = () => this.state.confirmationCode.length > 5 && this.state.newPassword.length > 5;
```

Comment:
We should look to validate the password in accordance with the User Pool password policy configuration. Can you create a separate issue for this please?